### ISSUE TITLE
Scroll inside child div of body 

### ISSUE THREAD

**Issue by georgecm** [usablica/intro.js#373]
I don't know if I use the plugin correctly, but I have tried it in several different pages with the same result.

The problem:

I have an absolute header and footer, and an absolute main "DIV". Inside this main DIV, I have a smaller one with overflow: auto and bigger height that the parent DIV, so I can have a scrollbar. 

The problem is that the plugin doesn't scroll inside this small scrollbar, but instead it keeps scrolling the body! So the end result is very bad.

Do I do something the wrong way, is this a known issue?

Thanks :)

**Comment by georgecm** [comment]
This seems a valid point, but the result was not good. The plugin somehow measured wrongly all the positions and misplaced the overlay and steps elements.

**Comment by pavankumarkatakam** [comment]
Yeah, i just want to take tour in a seperate div that acts as actual body here is the codepen http://codepen.io/pavankumarkatakam/pen/bedQdB,

**Comment by ludwich** [comment]
Would it be possible to add another optional parameter to introjs API for setting what to scroll ? As for now it always scrolls window but sometimes i want it to scroll in a div.

**Comment by renatograel14** [comment]
I found out a workaround, not sure if it applies for everyone. You can use Jquery, in this sample I'm using JQlite provided from angular

```js
var nativeElement = document.querySelector(selector); // current_step element
nativeElement.scrollIntoView(false); //scroll element to bottom of the parent
                    
var currentElement = angular.element(nativeElement);
if(currentElement) {
    var topOffset = currentElement.offset().top;

    //select helperlayer, tooltip and disableinteraction divs
    var disableInteractionLayer = angular.element(document.querySelector('body > div.introjs-disableInteraction')),
         tooltipLayer = angular.element(document.querySelector('body > div.introjs-tooltipReferenceLayer')),
         helperLayer = angular.element(document.querySelector('body > div.introjs-helperLayer'));

                            
    disableInteractionLayer.css('top', topOffset - 5 + 'px');
    tooltipLayer.css('top', topOffset - 5 + 'px');
    helperLayer.css('top', topOffset - 5 + 'px');
}
```

Let me know if it works for you, guys

**Comment by bozdoz** [comment]
Resolved with #801
